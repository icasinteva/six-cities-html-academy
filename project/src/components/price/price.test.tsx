import { render } from '@testing-library/react';

import Price from './price';

describe('Component: Price', () => {
  it('should render correctly', () => {
    const { container } = render(<Price className='property' amount={120} />);

    expect(container).toMatchSnapshot();
  });
});
