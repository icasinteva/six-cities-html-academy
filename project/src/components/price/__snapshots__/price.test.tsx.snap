// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Price should render correctly 1`] = `
<div>
  <div
    class="property__price"
    data-testid="property__price"
  >
    <b
      class="property__price-value"
    >
      â‚¬
      120
    </b>
    <span
      class="property__price-text"
    >
      <span
        class="price-divider"
        style="display: none;"
      >
        /
      </span>
      night
    </span>
  </div>
</div>
`;
